### DISCUSSION TITLE
Support Multiple Producers in Parallel Consumer?

### DISCUSSION THREAD

**Discussion by colinkuo** [confluentinc/parallel-consumer#461] [General]
Hello, 

The following class `ParallelConsumerOptions` only allows configuring a single producer. I wonder if we can configure multiple consumers in Parallel Consumer.

Thanks!

**Comment by astubbs** [comment]
Did you mean multiple producers? Or multiple consumers?

What's the use case? More capacity for sending large amounts of records concurrently? If so - sure, that makes sense. 

Any thoughts on how you'd like to see this? Round robin sending across the list of provided producers? 

We'd have to disable transaction support for this mode of operation, but I doubt anyone will object to that.
